processor/processor.go

```diff
@@ -2765,6 +2776,7 @@ func (proc *Handle) transformSrcDest(
 			proc.logger.Debug("Dest Transform input size", len(eventsToTransform))
 			s := time.Now()
 			response = proc.transformer.Transform(ctx, eventsToTransform, proc.config.transformBatchSize.Load())
+			proc.handleResponseForWarehouseTransformation(ctx, eventsToTransform, response, commonMetaData, eventsByMessageID)
```

Comment:
won't this run for all destinations? We should run this for only warehouse destinations rt

Reply:
Oops. Missed the warehouse destination check. Adding now.